coalib/settings/ConfigurationGathering.py

```diff
@@ -59,17 +59,18 @@ def load_config_file(filename, log_printer, silent=False):
     except FileNotFoundError:
         if not silent:
             if os.path.basename(filename) == Constants.default_coafile:
-                log_printer.warn('The default coafile {0!r} was not found. '
-                                 'You can generate a configuration file with '
-                                 'your current options by adding the `--save` '
-                                 'flag or suppress any use of config '
-                                 'files with `-I`.'
+                log_printer.warn('{0!r} was not found. '
+                                 'You can:\tGenerate a configuration file with '
```

Comment:
hmmm, interesting idea. what about:

Default coafile not found!
Here's what you can do:
   * add `--save` to generate a config file with your current options
   * add `-I` to suppress any use of config files

although with stars instead of bullet points